src/components/dashboard/Hub/Hub.tsx

```diff
@@ -1,13 +1,25 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { Typography } from '@material-ui/core'
 import ResponsiveStepWrapper from '../../common/ResponsiveStepWrapper'
-import { HubCardProps } from './Card'
+import Card, { HubCardProps } from './Card'
+import { useLocation } from 'react-router-dom'
 
 type Props = {
   cards: HubCardProps[]
 }
 
 function Hub({ cards }: Props) {
+  const [showEnrollment, setShowEnrollment] = useState(false)
+  const location = useLocation()
+  const params = new URLSearchParams(location.search)
+  const forceShowEnrollmentURLParam = params.get('forceShowEnrollment')
+
+  useEffect(() => {
```

Comment:
just wondering if we need the useEffect or even have `showEnrollment` in the state.  From the code it appears that the  only time forceShowEnrollmentURLParam gets set is when the components loads. If this is the case, won't 
` const showEnrollment = params.get('forceShowEnrollment') === 'true'` be sufficient?

Reply:
It sure does.  Thanks @conundrumgirl , I'll update the code for future releases...